env.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
```

Comment:
I still think this script is not necessary. (and no README, no one knows what it is)

Reply:
Make sense.